config/optional/core.entity_view_display.node.book.teaser.yml

```diff
@@ -23,7 +23,7 @@ content:
   field_book_image:
     type: image
     weight: 0
-    label: above
+    label: hide
```

Comment:
shouldn't this be hidden? 🤔

Reply:
@ronaldtebrake Not sure what happened there, fixed it :)